client/checker_client/cli.py

```diff
@@ -206,6 +206,7 @@ def checker(config: Config, checker_dir, oracle, ctez, token_metadata):
     checker = checker_lib.deploy_checker(
         client,
         checker_dir,
+        tez_wrapper=config.tez_wrapper_address,
```

Comment:
We should probably add an additional check similar to line 194 to show a friendly error message if the wrapper address isn't specified

https://github.com/tezos-checker/checker/pull/269/files#diff-65a5c024a0df2eb7077947147e98a51765d31b858c3a7c145fbd4d773cfc0063R194